src/Servers/Kestrel/Core/src/Internal/CertificatePathWatcherLoggerExtensions.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal;
+
+internal static partial class CertificatePathWatcherLoggerExtensions
+{
+    [LoggerMessage(1, LogLevel.Warning, @"Directory ""{Dir}"" does not exist so changes to the certificate ""{Path}"" will not be tracked", EventName = "DirectoryDoesNotExist")]
```

Comment:
Messages end with a fullstop.

Why `Dir` and not something like `Directory`. We usually don't have abbreviations. Or `WatchDirectory`?

Reply:
Also, most log messages use single quotes around values instead of double quotes.

Reply:
The punctuation I got from the file I copied, but I've updated it.

I've changed "Dir" to "Directory".  As a consumer, I think I'd want to consume directories as "Directory", rather than having to special case each log message.